List removed, deprecated, new options of the CREATE db cmd (#1312)
neo4j/docs-cypher: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

| Feature
| Details

a|
label:functionality[]
label:removed[]
[source, cypher, role="noheader"]
----
CREATE DATABASE db OPTIONS { seedCredentials: ... }
----

> | The option `seedCredentials` was removed from the `CREATE DATABASE` `OPTIONS` map.

Reviewer: ```suggestion
| The option `seedCredentials` is removed from the `CREATE DATABASE` `OPTIONS` map.
```

For more information, see link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/create-databases/#manage-databases-create-database-options[Operations Manual -> Database administration -> Create databases].

a|
label:functionality[]
label:removed[]
[source, cypher, role="noheader"]
----
CREATE DATABASE db OPTIONS { existingDataSeedInstance: ... }
----

> | The option `existingDataSeedInstance` was removed from the `CREATE DATABASE` `OPTIONS` map.

Reviewer: ```suggestion
| The option `existingDataSeedInstance` is removed from the `CREATE DATABASE` `OPTIONS` map and replaced by `existingDataSeedServer`.
```

> Use `existingDataSeedServer` instead.

Reviewer: ```suggestion
```

This allows a database to be seeded up to a specific date or transaction ID.
For more information, see link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/seed-from-uri/[Operations Manual -> Database administration -> Create a database from a URI].

a|
label:functionality[]
label:new[]
[source, cypher, role="noheader"]
----
CREATE DATABASE db OPTIONS { seedSourceDatabase: ... }
----

> | Now you can specify the name of a source database if the `seedURI` points to a folder containing backups for multiple databases.

Reviewer: ```suggestion
| You can specify the name of a source database if the `seedURI` points to a folder containing backups for multiple databases.
```
